src/js/footer/traderHubSignUpLanguageLinks/index.js

```diff
@@ -0,0 +1,44 @@
+import { getCookieByKey } from "../cookies";
+
+const allowedLangs = ["en", "fr", "de", "it", "es", "pt", "pl", "ru", "ar"];
```

Comment:
any possibility that this will be reused somewhere else? If yes let's move to constants

Reply:
no this is specific to the language support in traders hub this will be updated as tradershub adds more languages and eventually removed